src/components/datepicker/datePicker.scss

```diff
@@ -57,9 +85,11 @@ md-datepicker {
 
 // Floating pane that contains the calendar at the bottom of the input.
 .md-datepicker-calendar-pane {
+  // We need to pull the pane backwards, otherwise it
+  // causes an overflow in IE and Edge.
+  left: -100%;
```

Comment:
I'm confused about this line and the comment is IMO not saying why this is required.